net-vpn/protonvpn-cli/protonvpn-cli-2.2.0.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{3_6,3_7} )
```

Comment:
Upstream seems to support 3.8. Can you try enabling it?

Reply:
All right.  I have tested it with 3.6 and 3.7.  I don't have 3.8 enabled.  However, I will add it for testing.